spec/order_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "./lib/order.rb"
+
+describe Order do
+  subject(:order) {described_class.new}
+  let(:item1) {double(:item1)}
+  let(:item2) {double(:item2)}
+
+  before do
+      allow(item1).to receive(:name)       {"Salmon roll"}
+      allow(item1).to receive(:price)      {7.0}
+      allow(item1).to receive(:reference)  {"A1"}
+      allow(item2).to receive(:name)       {"Tuna roll"}
+      allow(item2).to receive(:price)      {8.5}
+      allow(item2).to receive(:reference)  {"B1"}
+  end
+
+  describe "#initialize" do
+
+    it "creates a blank array to store the ordered items in" do
+       expect(order.check_order.length).to eq 0
+     end
+
+  end
+
+  describe "#add_item to order_list" do
+
+    # it "can store ordered items in the order list" do
+    #   order.add_item(item1)
+    #   expect(order.check_order).to eq "#{item1.reference}....#{item1.name}....£#{item1.price}"
+    # end
+
+    it "can store more than one ordered items in the order list" do
+      order.add_item(item1)
+      order.add_item(item2)
+      expect(order.check_order).to eq [item1, item2] #"#{item1.reference}....#{item1.name}....£#{item1.price}\n#{item2.reference}....#{item2.name}....£#{item2.price}"
```

Comment:
Metrics/LineLength: Line is too long. [166/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/AsciiComments: Use only ascii symbols in comments. (https://github.com/bbatsov/ruby-style-guide#english-comments)